care/facility/models/patient_icmr.py

```diff
@@ -0,0 +1,198 @@
+        if instance is not None:
+            instance.__class__ = PatientSampleICMR
+        return instance
+
+    @property
+    def age_years(self):
+        if self.date_of_birth is None and self.year_of_birth is None:
+            age_years = self.age
+        elif self.year_of_birth is not None:
+            age_years = datetime.datetime.now().year - self.year_of_birth
+        else:
+            diff_days = (datetime.datetime.now().date() - self.date_of_birth).days
+            age_years = int(diff_days / 365)
+        return age_years
+
+    @property
+    def age_months(self):
+        if self.date_of_birth is None or self.year_of_birth is None:
+            age_months = 0
+        else:
+            diff_days = (datetime.datetime.now().date() - self.date_of_birth).days
+            age_years = int(diff_days / 365)
+            age_months = int((diff_days - age_years * 365) / 12)
+        return age_months
+
+    @property
+    def email(self):
+        return ""
+
+    @property
+    def pincode(self):
+        return ""
+
+    @property
+    def local_body_name(self):
+        return "" if self.local_body is None else self.local_body.name
+
+    @property
+    def district_name(self):
+        return "" if self.district is None else self.district.name
+
+    @property
+    def state_name(self):
+        return "" if self.state is None else self.state.name
+
+    @property
+    def has_travel_to_foreign_last_14_days(self):
+        return len(self.countries_travelled.strip()) != 0 and (
+            self.date_of_return and (self.date_of_return.date() - datetime.datetime.now().date()).days <= 14
+        )
+
+    @property
+    def travel_end_date(self):
+        return self.date_of_return.date()
+
+    @property
+    def travel_start_date(self):
+        return None
+
+    @property
+    def contact_case_name(self,):
+        contact_case = self.contacted_patients.first()
+        return "" if not contact_case else contact_case.name
+
+    @property
+    def was_quarantined(self,):
+        return None
+
+    @property
+    def quarantined_type(self,):
+        return None
+
+
+class PatientSampleICMR(PatientSample):
+    class Meta:
+        proxy = True
+
+    @property
+    def collection_date(self):
+        return self.date_of_sample.date()
+
+    @property
+    def label(self):
+        return ""
+
+    @property
+    def is_repeated_sample(self):
+        return None
+
+    @property
+    def lab_name(self):
+        return ""
+
+    @property
+    def lab_pincode(self):
+        return ""
+
+    @property
+    def hospitalization_date(self):
+        return self.consultation.admission_date.date()
+
+    @property
+    def medical_conditions(self):
+        return [
+            item.disease for item in self.patient.medical_history.all() if item.disease != DISEASE_CHOICES_MAP["NO"]
+        ]
+
+    @property
+    def symptoms(self):
+        return [
+            symptom
+            for symptom in self.consultation.symptoms
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Instance of 'ForeignKey' has no 'symptoms' member](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5346569)